nmostesting/suites/IS1001Test.py

```diff
@@ -225,9 +225,9 @@ def do_dns_sd_advertisement_check(self, test, api, service_type):
                         API protocol ('api_proto') TXT record is {} and not 'https'
                     """.format(properties["api_proto"]))
 
-                return test.PASS()
-        return test.FAIL("No matching mDNS announcement found for {} with IP/Port {}:{}."
-                         .format(api["name"], api["ip"], api["port"]))
+                return test.WARNING("Authorization Server SHOULD NOT be advertised by mDNS based DNS-SD")
+        return test.OPTIONAL("""No matching mDNS announcement found for {} with IP/Port {}:{}.
+                                This is recommended by IS-10.""".format(api["name"], api["ip"], api["port"]))

```

Comment:
Could you change 'this is recommended by' to something else? It sounds almost like you're saying you should use mDNS rather than suggesting that the 'Optional' result is actually fine.

I wonder if this should actually just be a PASS? Any thoughts @garethsb-sony? Otherwise if someone sees it's optional and implements it they're then going to get a warning back which looks worse.